build/stf-run-ci/tasks/create_builds.yml

```diff
@@ -37,10 +51,12 @@
   ansible.builtin.set_fact:
     build_describe: "{{ build_describe_results.stdout | from_json }}"
 
-- ansible.builtin.debug:
+- name: Show the ``oc get build ...`` results
```

Comment:
I know it's minor and doesn't really matter, but since this isn't RST I think it'd be better just to say "Get the Build results"